addons/collapse-footer/userscript.js

```diff
@@ -0,0 +1,16 @@
+export default async function ({ addon, console }) {
+  if (!addon.settings.get("infiniteScroll")) {
```

Comment:
We should check whether the addon is enabled in case the user disables infinite-scrolling, I think